_integration-schemas/adroll/v1/advertisables.md

```diff
@@ -0,0 +1,326 @@
+---
+tap: "adroll"
+version: "1"
+key: ""
+
+name: "advertisables"
+doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-advertisable-get"
+singer-schema: "https://github.com/singer-io/tap-adroll/blob/master/tap_adroll/schemas/advertisables.json"
+description: |
+  The `{{ table.name }}` table contains information about the advertisables in your {{ integration.display_name }} account.
+
+replication-method: "Full Table"
+
+api-method:
+    name: "getAdvertisables"
+    doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-advertisable-get"
```

Comment:
Similar issue to the endpoint in `ad_groups`, this isn't the endpoint the tap uses to get this data.